bridgetown-core/lib/site_template/package.json.erb

```diff
@@ -27,11 +27,12 @@
     <% if options["use-postcss"] %>
     "postcss": "^8.1.9",
     "postcss-flexbugs-fixes": "^4.1.0",
-    "postcss-loader": "^4.1.0",
+    "postcss-loader": "^5.0.0",
```

Comment:
I'd like to upgrade PostCSS but let's not include it in this PR as it'll require additional testing unrelated to Sass.

Reply:
Okay not sure how that happened but will push a fix.